lib/libmpa/mpa_random.c

```diff
@@ -35,12 +35,12 @@
 #include "mpa_assert.h"
 #include <tee_api_types.h>
 
-/* This function ends to the real RNG of the platform
- * Depending where the libmpa is used:
- * - kernel side: the implementation is found in the TEECore
- * - user side: the implementation is found in libutee
- */
-TEE_Result get_rng_array(void *buf, size_t blen);
+static random_generator_cb get_rng_array;
+
+void mpa_set_random_generator(random_generator_cb callback)
+{
+	get_rng_array = callback;
+}
```

Comment:
Your are right. Does it feasible to add mpa_set_random_generator(get_random_array) in TEE_BIgIntInit() or TEE_BigIntInitFMM() ?